src/test/java/seedu/address/model/task/DependencyGraphTest.java

```diff
@@ -27,17 +27,27 @@
     @Before
     public void setUp() {
         //Initializing task lists
-        Task a = new TaskBuilder().withName("A").build();
+        Task completed1 = new TaskBuilder().withName("E").withStatus(Status.COMPLETED).build();
+        Task completed2 = new TaskBuilder().withName("F").withStatus(Status.COMPLETED).build();
+        Task a = new TaskBuilder().withName("A").withDependency(completed1).withDependency(completed2).build();
         Task b = new TaskBuilder().withName("B").withDependency(a).build();
         Task c = new TaskBuilder().withName("C").withDependency(b).withDependency(a).build();
+        Task d = new TaskBuilder().withName("D").withDependency(c).withStatus(Status.COMPLETED).build();
```

Comment:
is there a reason why you named this one d and the other two completed tasks completed1 and completed2?